crystal/athena/Dockerfile

```diff
@@ -6,6 +6,8 @@ COPY src src
 COPY shard.yml ./
 COPY athena.yml ./
 
-RUN shards build --release --no-debug
+RUN shards build --production --no-debug
```

Comment:
@Blacksmoke16 what is the difference between this version and the previous one ?

Reply:
`Run in release mode. No development dependencies and strict sync between shard.yml and shard.lock.` From the shards repo.